MANIFEST.in

```diff
@@ -7,6 +7,7 @@ include README.rst
 include RELEASE.md
 include checklist.py
 include asv.conf.json
+include create_dev_env
```

Comment:
I think `check-manifest` only compares source to the dist. It doesn't really have any reason to look at the manifest.